website/versioned_docs/version-v3.17.x/install.md

```diff
@@ -28,7 +28,7 @@ For either installation method, make sure you have cluster admin permissions:
 If you want to deploy a released version of Gatekeeper in your cluster with a prebuilt image, then you can run the following command:
 
 ```sh
-kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/v3.17.0/deploy/gatekeeper.yaml
+kubectl apply -f https://raw.githubusercontent.com/open-policy-agent/gatekeeper/v3.17.1/deploy/gatekeeper.yaml
```

Comment:
should we also release 3.18.0-beta.1 otherwise the latest doc will still point to 3.18.0-beta.0?

Reply:
I am onboard with releasing 3.18.0-beta.1.